### DISCUSSION TITLE
run error

### DISCUSSION THREAD

**Discussion by darelikz** [charliedream1/ai_quant_trade#5] [Q&A]
![image](https://github.com/charliedream1/ai_quant_trade/assets/27019179/8a4e644e-59a9-4c1f-a44f-46eebb9fc4bd)
file path :   ....\ai_quant_trade-master\egs_trade\rl\a001_proto_sb3\main.py

**Comment by charliedream1** [comment]
看着像是您的输入数据有问题，输入的tensor为nan了。你按照我的那个给出的数据准备流程，先用那个同样的情况准备一下数据，看看能否跑通，然后再换自己的数据试试呢



---Original---
From: ***@***.***&gt;
Date: Fri, Jun 30, 2023 14:24 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [charliedream1/ai_quant_trade] run error (Discussion #5)




 

 file path :   ....\ai_quant_trade-master\egs_trade\rl\a001_proto_sb3\main.py
